2_pvc_destination_gen/README.md

```diff
@@ -0,0 +1 @@
+TODO
```

Comment:
Basic doc like https://github.com/konveyor/pvc-migrate/blob/master/1_pvc_data_gen/README.md would be great if possible

Reply:
@djwhatle added, PTAL